playwright-e2e/tests/dsm/participant-page/participant-page-tag.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { expect, test } from '@playwright/test';
+import { login } from 'authentication/auth-dsm';
+import { study } from 'pages/dsm/navbar';
+import Select from 'lib/widget/select';
+import ParticipantsPage from 'pages/dsm/participants/participants-page';
+import Table from 'lib/widget/table';
+import Checkbox from 'lib/widget/checkbox';
+import { Page } from '@playwright/test';
+
+
+test.describe('Participant Page DSM', () => {
+    test.beforeEach(async ({ page }) => {
+      await login(page);
+    });
+      
+    test('Ensure cohort tags update and delete properly for Brain @dsm @dsm-search', async ({ page }) => {
+      await cohortTagTest("Brain", page)
+    });
+  
+    test('Ensure cohort tags update and delete properly for PanCan @dsm @dsm-search', async ({ page }) => {
+      await cohortTagTest("PanCan", page)
+    });
+  
+  });
+
+async function cohortTagTest(studyName: string, page: Page) {
+    //Get current date and time to ensure tag is different from others
+    const currentdate = new Date(); 
+    const dateString = (currentdate.getMonth()+1)+ "/"
+                + currentdate.getDate()  + "/" 
+                + currentdate.getFullYear() + "@"  
+                + currentdate.getHours() + ":"  
+                + currentdate.getMinutes() + ":" 
+                + currentdate.getSeconds();
+    //Go to study
+    await new Select(page, { label: 'Select study' }).selectOption(studyName);
+    await expect(page.locator('h1')).toHaveText('Welcome to the DDP Study Management System');
+    await expect(page.locator('h2')).toHaveText('You have selected the ' + studyName+ ' study.');
+ 
+    await study(page).selectOption('Participant List', { waitForNav: true });
+    await expect(page.locator('h1')).toHaveText('Participant List', { timeout: 30 * 1000 });
+ 
+    const participantListPage = new ParticipantsPage(page);
+    await participantListPage.waitForReady();
+ 
+    //Filter to only patients with medical records
+    await participantListPage.filterMedicalRecordPts();
+    await participantListPage.waitForReady();
+    const table = new Table(page);
+    //Make sure we got at least one result and click the first result.
+    expect(await table.rowLocator().count()).toBeGreaterThanOrEqual(1);
+    const cell = await table.cell(0, 2)
+    await cell.click();
+    await expect(page.locator('h1')).toHaveText('Participant Page', { timeout: 5 * 1000 });
+ 
+    //Make new tags
```

Comment:
can we replace the `_rgp` and `RGP` with the study name from the test `PanCan` or `Brain`?

Reply:
Good suggestion! Done 👍